--- git status ---
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   isaaclab/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/grallator/__pycache__/grallator.cpython-311.pyc
	modified:   isaaclab/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/grallator/grallator.py

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	isaaclab/analyze_rl_csv_logs.py
	isaaclab/export_rl_logs_to_csv.py
	isaaclab/logs/rsl_rl/anymal_c_flat/
	isaaclab/logs/rsl_rl/grallator_flat/2026-05-16_11-36-14/
	isaaclab/logs/rsl_rl/grallator_flat/2026-05-16_11-37-23/
	isaaclab/logs/rsl_rl/grallator_flat/2026-05-16_11-38-11/
	isaaclab/logs/rsl_rl/grallator_flat/2026-05-16_11-38-50/
	isaaclab/logs/rsl_rl/grallator_flat/2026-05-16_11-39-45/
	isaaclab/logs/rsl_rl/unitree_go2_flat/
	isaaclab/outputs/2026-05-16/
	isaaclab/rl_log_exports/
	isaaclab/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/anymal_c/__pycache__/flat_env_cfg.cpython-311.pyc
	isaaclab/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/anymal_c/__pycache__/rough_env_cfg.cpython-311.pyc
	isaaclab/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/anymal_c/agents/__pycache__/rsl_rl_ppo_cfg.cpython-311.pyc
	isaaclab/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/go2/__pycache__/flat_env_cfg.cpython-311.pyc
	isaaclab/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/go2/__pycache__/rough_env_cfg.cpython-311.pyc
	isaaclab/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/go2/agents/__pycache__/rsl_rl_ppo_cfg.cpython-311.pyc
	isaaclab/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/mdp/symmetry/__pycache__/
	isaaclab/test_grallator_crouch_stand_walk.py
	isaaclab/test_grallator_pd_trot_walk.py

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/isaaclab/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/grallator/__pycache__/grallator.cpython-311.pyc b/isaaclab/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/grallator/__pycache__/grallator.cpython-311.pyc
index c383403..625f00d 100644
Binary files a/isaaclab/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/grallator/__pycache__/grallator.cpython-311.pyc and b/isaaclab/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/grallator/__pycache__/grallator.cpython-311.pyc differ
diff --git a/isaaclab/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/grallator/grallator.py b/isaaclab/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/grallator/grallator.py
index 559c061..abff665 100644
--- a/isaaclab/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/grallator/grallator.py
+++ b/isaaclab/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/grallator/grallator.py
@@ -67,9 +67,9 @@ GRALLATOR_CFG = ArticulationCfg(
             ],
             effort_limit=60.0,
             saturation_effort=60.0,
-            velocity_limit=2.0,
-            stiffness=200.0,
-            damping=10.0,
+            velocity_limit=20.0,
+            stiffness=80.0,
+            damping=3.0,
             friction=0.0,
         ),
     },